### PR TITLE
Updated kafka consumer to properly use binary/structured mode

### PR DESCRIPTION
**Pull request by mmacai**
- [x] Update docs

## Description

- updated kafka consumer to properly use binary/structured mode
- updated Kafka headers serializer to use strings instead of atoms

## What to look out for

Dear reviewer, I want you to

- gain knowledge - I think it's important for you to know about the change.
- check that the code works on your machine.
- suggest implementation-code design/structure/readability improvements.
- suggest test-code design/structure/readability improvements.
- suggest documentation improvements.
- suggest language/writing improvements.
- help to find bugs (described below!).

## Process

The goal is to improve not only the code in this PR but also our skills! The "rules":

- The review is considered "done" as soon as all reviewers have added their review, and all their comments have been addressed.
- For knowledge-sharing reviews, each reviewer should "approve" the PR after studying its content.
- After the approval, the merge is concluded by the developer.

Have fun!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,14 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Support publishing events consumed from [NATS](https://nats.io) topics. See the [documentation](https://accenture.github.io/reactive-interaction-gateway/docs/event-streams.html#nats) for how to get started.
-  [#297](https://github.com/Accenture/reactive-interaction-gateway/issues/297)
-- Added possibility to define Kafka/Kinesis topic and schema per reverse proxy endpoint. The current solution using environment variables is deprecated, but still used as a fallback -- will be removed in the version 3.0.
-  [#229](https://github.com/Accenture/reactive-interaction-gateway/issues/229)
-- Added Kinesis + Localstack example.
-  [#229](https://github.com/Accenture/reactive-interaction-gateway/issues/229)
```
Comment (kevinbader) [inline CHANGELOG.md:17]: removed in error?

Comment (mmacai) [reply CHANGELOG.md:17]: Actually on purpose. Seems like we did wrong merge in Nats PR as these logs are part of the 2.4 version.

**Comment by mmacai** [comment]
@kevinbader updated docs a bit according to changes and fixed some npm audit issues.

**Comment by mmacai** [comment]
Closing in favour of https://github.com/Accenture/reactive-interaction-gateway/pull/308.

**Pull request closed without merging**